kernel/src/user_program/elf_loader.rs

```diff
@@ -0,0 +1,133 @@
+extern crate core;
+
+use core::slice;
+use virtual_memory_area::VmAreaStruct;
+use virtual_memory_area::VmFlags;
```

Comment:
```suggestion
use super::virtual_memory_area::{VmAreaStruct, VmFlags};
```

The first two statements are unnecessary, and the second two need the `super::` prefix and can be combined.